### DISCUSSION TITLE
List of services backup stops/doesn't stop

### DISCUSSION THREAD

**Discussion by Aevrin** [MichaIng/DietPi#6640] [Q&A] [answered]
I wanted to confirm if there's a list of services that are stopped, or not stopped during a backup?  My plan is to use docker to provide many services, including some not in dietpi, but I want to make sure that backup stops docker so I can backup any volumes.  Then I can have syncthing sync the backups to an offsite instance.

**Comment by Joulinar** [comment, marked as answer]
following services will be stopped automatically during backup https://github.com/MichaIng/DietPi/blob/master/dietpi/dietpi-services#L115

And some special we don't touch https://github.com/MichaIng/DietPi/blob/master/dietpi/dietpi-services#L279

**Reply by Aevrin** [reply]
Thanks, I had gotten lost looking in things and missed the obvious pointers (not to mention that I don't code except for fun).

**Comment by MichaIng** [comment]
You can also run this command to see which services are started (and hence also stopped at first), which is exactly what `dietpi-backup` does as well:
```sh
dietpi-services start
```
Note that Docker containers can be configured to not stop with the Docker daemon itself, but by default they are.

**Reply by Aevrin** [reply]
I don't have a currently running install, I decommissioned my last two before I moved everything to docker and use omv and vanilla Oracle Ubuntu.  I'm looking at moving back to dietpi, and was preparing my way.
